### ISSUE TITLE
In AR Mode, the model is not on ground

### ISSUE THREAD

**Issue by Qiweih** [google/model-viewer#3164]
### Description
Hello, 

I encounter a problem by opening my model in AR mode, when i open the ar mode in mobile device, the model is not on ground after detecting ground.

Here is a link to the model : https://xrshow.tech:8445/upload/test3d/0.glb

### Version

- model-viewer: v1.10.1

### Browser Affected

- [ x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [x ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [x ] WebXR
- [ ] SceneViewer
- [ ] QuickLook

**Comment by samaneh-kazemi** [comment]
Hi, can you please share a link to your code so we can try it? What phone you are using?

**Comment by Qiweih** [comment]
I didn't add any special settings, all are default settings, I'm using a XIAOMI 9 device. I tried the default model Astronaut, and it's working, the model is on ground in AR mode, but not with my model. 
However for my model, the origin point is in (0,0,0) too, i do not understand why it's flying. (About 1 m from the ground)

**Comment by Qiweih** [comment]
https://speckle-jet-sardine.glitch.me/

Here is a glitch link

**Comment by samaneh-kazemi** [comment]
Thanks for sharing. Is your glitch link supposed to show the astronaut for both poster and AR model?

**Comment by Qiweih** [comment]
@samaneh-kazemi On my side, i'm seeing my AR model (https://xrshow.tech:8445/upload/test3d/0.glb) 
But the poster is astronaut yes.

**Comment by samaneh-kazemi** [comment]
Thanks for sharing the link. I was able to download it and upload it in the [editor](https://modelviewer.dev/editor/) successfully. @elalish I'm not seeing this issue in iPhone. But can you check it for Android as it's stated in the bug?
@Qiweih can you please share which phone and android version are you using?

**Comment by samaneh-kazemi** [comment]
@Qiweih Btw, once you upload the model in our editor, if you download the scene or poster, it will autogenerates the poster for you.

**Comment by Qiweih** [comment]
@samaneh-kazemi that's a nice feature for the poster ! 
The bug is in AR mode, but when i target the camera in (0,0,0) i see that the model is not in (0,0,0) position. I'm checking the gltf file in order to see what is the problem as in blender it was in (0,0,0) position.

**Comment by Qiweih** [comment]
@samaneh-kazemi I don't know if model viewer has the ability to put my model in (0,0,0) position ?

**Comment by Qiweih** [comment]
@samaneh-kazemi I solve the problem, the model was too big and this problem appear, i scale it down and now it's ok thanks for the help

**Comment by Narendra186** [comment]
> @samaneh-kazemi I solve the problem, the model was too big and this problem appear, i scale it down and now it's ok thanks for the help

Hi, I'm facing this issue too. When you spoke of scale down, do you mean it in terms of file size or model dimensions?

**Comment by elalish** [comment]
I would bet physical scale (glTF is always in meters, and many programs export them without asking what units you were modeling in). You can try it quickly with https://modelviewer.dev/docs/#entrydocs-scenegraph-attributes-scale

**Comment by Narendra186** [comment]
@elalish I got a good result by scaling them physically. I would like to discuss a problem I have been facing for a long time.
My models rendered using WebXR mode are not of real world dimensions in AR. Could this be a problem on WebXR side or is  there any way to configure that behaviour in model viewer?

**Comment by elalish** [comment]
Please open a new discussion with a repro link; it's hard to follow closed threads. WebXR will show objects at true scale, keeping in mind that glTF dimensions are meters. If ARCore hasn't found the floor yet, then it may be further away than the floor, thus appearing small. If you move side-to-side you'll notice the difference between placement and scale.
